Alexandr_hw_4.js

```diff
@@ -0,0 +1,165 @@
+// ## Complete exercise
+
+// ### NORMAL level
+
+// #### Task 1 💻
+
+// function getSum (con){
+//     let sum = 0;
+//     for (let i = 0; i <= con; i++){
+//         sum = sum + i;
+//     }
+//     return sum
+// }
+// console.log(getSum(100));
+
+// #### Task 2 💻
+
+// const PERSENT = 17;
+// const AGE = 5;
+// function sumCredit (credit){
+//     let plusCredit = AGE * ((credit * 17) / 100);
+//     console.log(plusCredit);
+
+// }
+// sumCredit(50);
+
+// #### Task 3 💻
+
+// const gh = "Привет я Саша я живу в Сан-Франциско";
+// function trimString(str, start, end) {
+//   let newStr = "";
+//   for (let i = 0; i < str.length; i++) {
+//     if (str[i].toLowerCase() === start.toLowerCase()) {
+//       while (str[i] !== end) {
+//         newStr += str[i];
+//         i++;
+//       }
+//       newStr += end;
+//       return newStr;
+//     }
+//   }
+// }
+// console.log(trimString(gh, "Ф", "с"));
+
+// #### Task 4 💻
+
+// function getSumNumbers(jum){
+//     let jumString = jum.toString();
+//     let sumJum = 0;
+//     for(let i = 0; i < jumString.length; i++){
+//         sumJum += Number(jumString[i]);
+//     }
+//     return sumJum;
+// }
+// console.log(getSumNumbers(2021));
+
+// #### Task 5 💻
+
+// function getSum(a, b){
+//     if (a == b){
+//         return a;
+//     }
+//     else if (a > b){
+//         return a +b;
+//     }
+//     else {
+//         let sumAB = 0;
+//         for (let i = a; i <= b; i++){
+//             sumAB += i;
+//         }
+//         return sumAB;
+//     }
+// }
+// console.log(getSum(-1, 2));
+
+// #### Task 6 💻
+
+// const foo = function(){
+//     return "Name FOO";
+// }
+// const boo = function(){
+//     return "Name BOO";
+// }
+
+// function fooboo (tubel, floo, bloo){
+//     if (tubel == true){
+//         return floo();
+//     }
+//     else {
+//         return bloo();
+//     }
+// }
+// console.log(fooboo(true, foo, boo));
+
+// ### ADVANCED level
+
+// #### Task 1 👨‍🏫
+
+// function triangle (a, b, c) {
+//     if (a + b < c || a + c < b || b + c < a || a < 0 || b < 0 || c < 0){
```

Comment:
here you can just return (a + b < c || a + c < b || b + c < a || a < 0 || b < 0 || c < 0); cause it is boolean and you trying to return boolean